x/superfluid/keeper/migrate_test.go

```diff
@@ -377,7 +437,7 @@ func (suite *KeeperTestSuite) TestMigrateSuperfluidUnbondingBalancerToConcentrat
 			}
 
 			// System under test.
-			positionId, amount0, amount1, _, _, newGammLockId, concentratedLockId, poolIdLeaving, poolIdEntering, err := superfluidKeeper.MigrateSuperfluidUnbondingBalancerToConcentrated(ctx, poolJoinAcc, originalGammLockId, coinsToMigrate, synthLockBeforeMigration[0].SynthDenom, tc.tokenOutMins)
+			positionId, amount0, amount1, _, _, concentratedLockId, poolIdLeaving, poolIdEntering, err := superfluidKeeper.MigrateSuperfluidUnbondingBalancerToConcentrated(ctx, poolJoinAcc, originalGammLockId, coinsToMigrate, synthLockBeforeMigration[0].SynthDenom, tc.tokenOutMins)
 			if tc.expectedError != nil {
```

Comment:
ditto here for additional checks

Reply:
I expanded checks for the route and delegating. For the undelegating checks though, what should be expanded? Since, with superfluid, once something is undelegating, it is instantly undelegated. So if something is in an undelegating status prior to migration, I am not sure what extra things we should check here.

Reply:
Checked the exact amounts here https://github.com/osmosis-labs/osmosis/pull/5162/commits/c20ddedc31c5d790d0baee6d6df3fc8314f163a3